src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+}
+
+.nav__logo {
+  padding-left: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav .nav__list {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
```

you don't need it for `nav` nav only has one child
```suggestion
.nav__list {
  display: flex;
  align-items: center;
  margin: 0;
  padding: 0;
}
```